crates/rooch-key/src/keystore.rs

```diff
@@ -116,6 +120,28 @@ impl<'de> Deserialize<'de> for FileBasedKeystore {
     }
 }
 
+// #[cfg(any(test, feature = "fuzzing"))]
```

Comment:
should set the features = ["fuzzing"] on the rooch-types dependency "rooch-types = { workspace = true }"